gems/aws-sdk-core/lib/aws-sdk-core/plugins/regional_endpoint.rb

```diff
@@ -47,21 +43,27 @@ class RegionalEndpoint < Seahorse::Client::Plugin
       end
 
       def after_initialize(client)
-        if client.config.region.nil? or client.config.region == ''
+        if client.config.region.nil? || client.config.region == ''
           raise Errors::MissingRegionError
         end
+
+        # check region format follows pattern <continent>-<direction>-<number>
```

Comment:
The comment doesn't quite match the check that we're doing.  We're only checking for a valid label/sub-domain format and normally aws regions follow the <continent>-<direction>-<number> format.

Reply:
what's a good comment I can put here for my future self?